src/BorderBox.js

```diff
@@ -8,7 +8,8 @@ const BorderBox = styled(Box)(BORDER)
 
 BorderBox.defaultProps = {
   theme,
-  border: '1px solid',
+  borderWidth: '1px',
+  borderStyle: 'solid',
```

Comment:
Should we update our theme accordingly? Right now it's:

`const borders = [0, '1px solid']`

maybe it should look more like this now:

```
borders: {
  default : {
   width: '1px',
   style: 'solid',
   color: 'gray.2'
 }
}
```

Reply:
I asked in Slack if there's any others that we should support: https://github.slack.com/archives/GACAW0NPM/p1587406624466900

Not sure if this is the best API for the theme but something to get started with

Reply:
The theme object for borders should probably include everything here: https://primer.style/css/utilities/borders#border-width-style-and-color-utilities

This is fine for a follow up PR but I'd love to get this in the same release since it's a breaking change!

Reply:
@emplums How about:

```javascript
const borders = {
  light: {
    width: '1px',
    style: 'solid',
    color: colors.border.grayLight
  },
  normal: {
    width: '1px',
    style: 'solid',
    color: colors.border.gray
  },
  dark: {
    width: '1px',
    style: 'solid',
    color: colors.border.grayDark
  }
}
```